docker/run-setup.php

```
@@ -321,6 +321,54 @@ function callModuleInstall(string $fullModulePath): void {
+ * Important: Keep the event parameters in sync with install() in:
+ * src/admin/controller/module/ukaddresssearch.php
+ */
+function registerEvent(): void {
+    global $registry;
+    $db = $registry->get('db');
+    
+    echo "Registering ukaddresssearch event...\n";
+    
+    try {
+        // Check if event already exists
+        $query = $db->query("SELECT * FROM `" . DB_PREFIX . "event` WHERE `code` = 'ukaddresssearch'");
```

PHPStan (2.1.55)

[error] Constant DB_PREFIX not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)